MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryAddCategoryImageTest.xml

```diff
@@ -19,10 +19,12 @@
         </annotations>
         <before>
             <createData entity="SimpleSubCategory" stepKey="category"/>
+            <magentoCLI command="config:set cms/wysiwyg/enabled enabled" stepKey="enableWYSIWYG"/>
```

Comment:
the `AdminEnableWYSIWYGActionGroup/AdminDisableWYSIWYGActionGroup` calls exactly the came command. What is the reason of calling the `magentoCLI` instead of the corresponding action group?

Also, I think enabling/disabling wysiwyg can be removed from the test at all as it is handled by the test suite.

Reply:
Since Page Builder extends AdminEnableWYSIWYGActionGroup and enables Page builder, this test failing when running builds with Page Builder. As far as I see this test intentionally enables WYSIWYG.  So my suggestion is to enable and disable it directly with CLI command

Reply:
@omiroshnichenko do you have the issue only with this single ASI test?

Reply:
`MediaGalleryUiSuite` is disabling the WYSIWYG first as I believe it's enabled by default.

I think it's better to introduce the test suite that does not disable WYSIWYG and change the group of this test (while removing the enable/disable wysiwyg from the test itself) if that will fix the problem